README.md

```diff
@@ -1,58 +1,95 @@
+Please send issues and patches.
+
+The libraries can be used two ways:
+- either you can specify configuration parameters in the process dictionary. Useful for simple tasks
+- you can create a configuration object and pass that to each request as the final parameter. Useful for Cross AWS Account access
 
-All API functions have been implemented.  Not all functions have been thoroughly tested, so exercise care when integrating this library into production code.  Please send bug reports and patches.
+## Roadmap ##
 
-The libraries can be used two ways: either you can specify configuration parameters in the process dictionary, or you can create a configuration object and pass that to each request as the final parameter.
+Below is the proposed library roadmap update along with regular features and fixes.
 
-## Getting started ##
-You need to clone the repository and download rebar (if it's not already available in your path).
+- v0.13.9
+ * Existing code
 
-```
-git clone https://github.com/erlcloud/erlcloud.git
-cd erlcloud
-wget http://cloud.github.com/downloads/basho/rebar/rebar && chmod u+x rebar
-```
+- v2.0.X
+ * Going further we would like to merge in [Alert Logic](https://github.com/alertlogic/erlcloud/tree/v1.2.3) fork.
+ This is a major version bump which contains lots of new features and functionality.
+ Unfortunately, it also contains quite a number of low level APIs incompatibilities since the fork diverged for a long while.
+ No APIs have been removed and it's on top or 0.13.8 at the moment.
+ Making it backward compatible does not seem feasible and valuable at the moment.
 
-You can provide your amazon credentials in environmental variables.
+- v2.1.X
+ * fix dialyzer findings and make it mandatory for the library
+ * make full support of Mix/HEX
 
+- v2.2.X
+ * Further deprecation of legacy functionality
+  * Only SigV4 signing and generalised in one module. Keep SigV2 in SBD section only
+  * no more `erlang:error()` use and use of regular tuples as error API.
+
+## Getting started ##
+You need to clone the repository and download rebar/rebar3 (if it's not already available in your path).
 ```
-export AWS_ACCESS_KEY_ID=<Your AWS Access Key>
-export AWS_SECRET_ACCESS_KEY=<Your AWS Secret Access Key>
+git clone https://github.com/erlcloud/erlcloud.git
+cd erlcloud
+wget https://s3.amazonaws.com/rebar3/rebar3
+chmod a+x rebar3
 ```
 To compile and run erlcloud
 ```
 make
 make run
 ```
 
-If you're using erlcloud in your application, add it as a dependency in your application's configuration file.  To use erlcloud in the shell, you can start it by calling:
+If you're using erlcloud in your application, add it as a dependency in your application's configuration file.
+To use erlcloud in the shell, you can start it by calling:
 
 ```
-ssl:start().
-erlcloud:start().
+application:ensure_all_started(erlcloud).
 ```
+You can provide your amazon credentials in environmental variables.
```

Comment:
Is it worth also mentioning the support for profiles added by @fp here? That stuff is super cool.